File: P3-ontologija-API-sval/src/Svalina/Ontologija.php
<?php

namespace Svalina;

/**
 * @Entity @Table(name="ontologija")
 **/


class Ontologija 

{
    /** @id @Column(type="integer") @GeneratedValue **/
    protected $sifra;

    /**
    * @Column(type="string")
    */
    private $film;

     /**
    * @Column(type="string")
    */
    private $glumac;

     /**
    * @Column(type="string")
    */
    private $zanr;

     /**
    * @Column(type="string")
    */
    private $nagrada;

     /**
    * @Column(type="integer")
    */
    private $godinaIzlaska;

    /**
    * @Column(type="string")
    */
    private $trajanje;


    

    public function getSifra(){
      return $this->sifra;
    }
  
    public function setSifra($sifra){
      $this->sifra = $sifra;
    }
  
    public function getFilm(){
      return $this->film;
    }
  
    public function setFilm($film){
      $this->film = $film;
    }
  
    public function getGlumac(){
      return $this->glumac;
    }
  
    public function setGlumac($glumac){
      $this->glumac = $glumac;
    }
  
    public function getZanr(){
      return $this->zanr;
    }
  
    public function setZanr($zanr){
      $this->zanr = $zanr;
    }
  
    public function getNagrada(){
      return $this->nagrada;
    }
  
    public function setNagrada($nagrada){
      $this->nagrada = $nagrada;
    }
  
    public function getGodinaIzlaska(){
      return $this->godinaIzlaska;
    }
  
    public function setGodinaIzlaska($godinaIzlaska){
      $this->godinaIzlaska = $godinaIzlaska;
    }
  
    public function getTrajanje(){
      return $this->trajanje;
    }
  
    public function setTrajanje($trajanje){
      $this->trajanje = $trajanje;
    }
    

  
    public function setPodaci($podaci)
    {
      foreach($podaci as $kljuc => $vrijednost){
        $this->{$kljuc} = $vrijednost;
      }
    }
    
  }

?>

File: P3-android-app/app/src/main/java/ffos/p3/ontologija/Ontologija.java
package ffos.p3.ontologija;

import java.io.Serializable;

public class Ontologija implements Serializable {

    public Ontologija() {
        this.sifra = sifra;
        this.film = film;
        this.glumac = glumac;
        this.zanr = zanr;
        this.nagrada = nagrada;
        this.godinaIzlaska = godinaIzlaska;
        this.trajanje = trajanje;
    }

    private int sifra;
    private String film;
    private String glumac;
    private String zanr;
    private String nagrada;
    private int godinaIzlaska;
    private String trajanje;

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public String getFilm() {
        return film;
    }

    public void setFilm(String film) {
        this.film = film;
    }

    public String getGlumac() {
        return glumac;
    }

    public void setGlumac(String glumac) {
        this.glumac = glumac;
    }

    public String getZanr() {
        return zanr;
    }

    public void setZanr(String zanr) {
        this.zanr = zanr;
    }

    public String getNagrada() {
        return nagrada;
    }

    public void setNagrada(String nagrada) {
        this.nagrada = nagrada;
    }

    public int getGodinaIzlaska() {
        return godinaIzlaska;
    }

    public void setGodinaIzlaska(int godinaIzlaska) {
        this.godinaIzlaska = godinaIzlaska;
    }

    public String getTrajanje() {
        return trajanje;
    }

    public void setTrajanje(String trajanje) {
        this.trajanje = trajanje;
    }
}
